examples/wasm/file_system.go

```diff
@@ -1,28 +0,0 @@
-package main
```

Comment:
This doesn't work even with Wasmtime. So removing for now to not only prioritize the benchmarks, but also we will not focus on WASI until we finish JIT.

Reply:
yep pick battles!